src/examples/src/App.tsx

```diff
@@ -0,0 +1,108 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import block from '@dojo/framework/core/middleware/block';
```

Comment:
What does this block middleware do?

Reply:
There is some information here in the docs https://dojo.io/learn/building/buildtime-rendering#dojo-blocks